docs/understanding-radicle/how-radicle-works.md

```diff
@@ -0,0 +1,334 @@
+RAD_](https://radicle.mirror.xyz/CgcHpSXUlPvwMVaUVVaJ7r8bIJI2BOKOytaI9-nO9oY), which details the governance token used
+by the DAO.
+
+However, for the sake of this document, we'll focus on the user-facing components of this ecosystem &mdash; what you can
+interact with, as a user, to securely collaborate on software over a peer-to-peer network built on Git.
+
+## Key components of the Radicle ecosystem
+
+There are a few important open-source protocols, concepts, projects, and assets that integrate to make [Radicle's
+vision](https://radicle.mirror.xyz/Qz4Rr0R54KLPurqqswsQ2tfRtBs9MOYpw3dlY9VjyQ8) for decentralized code collaboration
+possible.
+
+- [**Identity**](#radicle-identity): A cryptographically-signed JSON document that identifies a device or a project,
+  stored alongside standard Git data when hosted, cloned, and replicated.
+- [**Network**](#radicle-network): The people who use Radicle tooling and the projects they collaborate on, supported by
+  seed nodes, which provide a data availability layer for projects and identities.
+- [**Peer-to-peer protocol**](#peer-to-peer-protocol): The protocol for disseminating Git repositories via gossip-based
+  replication, enabling the hosting and sharing of projects without knowing where it's physically stored within the
+  Radicle network.
+- [**Command-line tooling (`rad`)**](#command-line-tooling-rad): The core tool for managing your code, project, and
+  identity.
+- [**Web app**](#web-app): The web-based interface for discovering, viewing, and sharing Radicle projects.
+- [**Open-source funding (Drips)**](#open-source-funding-drips): An Ethereum protocol for continuously streaming funds
+  to others, which will be integrated into the [web app](#web-app) as new value flows for developers to sustain their
+  FOSS work.
+
+## Radicle identity
+
+Everything Radicle is trying to build around decentralized code collaboration starts with the concept of identity. In
+Radicle, your identity is a *portable* JSON document that defines a user or a project on the Radicle network.
+
+The most important part is the hashed content identifier—the **peer ID**—which identifies it uniquely wherever it's
+stored or discovered on the Radicle network. Each Radicle identity also has a few required metadata fields, which
+determine whether it's a user or project identity:
+
+```rust
+struct User {
+    /// A short name (nickname, handle), without any prefix such as the `@`
+    /// character
+    name: String,
+}
+
+struct Project {
+    /// A short name
+    name: String,
+
+    /// A slightly longer description (should fit in a headline)
+    description: Option<String>,
+
+    /// The default branch. "master" is assumed for git repositories if
+    /// unspecified.
+    default_branch: Option<String>,
+}
+```
+
+An identity can also be extended with additional metadata as needed by the client or interface that's interacting with
```

Comment:
What's the plan for more tutorial-like documentation?

To me, it would make sense to link to "How to create your first person and project" above and then "How to extend your metadata" here.

Reply:
I'll probably start planning as soon as this PR gets merged :)

Is there any current practical use for extending metadata right now? This sentence is pulled from an old @cloudhead presentation on identity, so it might not be super relevant any more. Are issues considered metadata for a project's identity?

Reply:
> Is there any current practical use for extending metadata right now? This sentence is pulled from an old @cloudhead presentation on identity, so it might not be super relevant any more. Are issues considered metadata for a project's identity?

The idea for extending metadata is to allow application developers to add their own data about a `Project` or `Person` and build on top of those extensions. For example, they could add an Ethereum address (which I think was done at some point?). So TL;DR, there are infinite practical uses, as long as you can imagine it :)